app/Models/Hit.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models;
+
+use Carbon\CarbonImmutable;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Hit extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['endpoint', 'user_agent'];
+
+    public static function forTimePeriod(string $period = 'week')
```

Comment:
I think it might be worth adding a docBlock here (and you know I hate docBlocks!) with either inline documentation or a link to documentation about what period strings we can use with carbon/datetime.